Code Update - Bind Azure Broker Service (SQL & MySQL)

ecointet opened the pull request:
New Database connection code to use Azure Services with Pivotal Azure Broker Services.
The .NET App is now ready to demo PAS with MS databases (Azure SQL or Azure for MySQL).

ecointet committed:
Code change - Azure Broker Service (SQL & MySQL)

New Database connection to use Azure Services with Pivotal Azure Broker Services.
Ready with Azure SQL or Azure MySQL.

rossr3-pivotal commented:
Instead of creating a branch, can you make the code smarter by having it detect both scenarios, rather than creating a branch?

ecointet committed:
Multidatabases support

Update with a smarter code to support multidatabases (MySQL for PCF, Mysql for Azure, and MS SQL for Azure).

ecointet commented:
Done! I updated the code to support the following scenarios:
1) MySQL for PCF
2) MySQL for Azure (azure broker)
3) MS SQL for Azure (azure broker)

Tested on PCF 2.0, on Azure.
